OpenSim/Common/Storage.cpp

```diff
@@ -3328,23 +3328,19 @@ double Storage::compareColumnRMS(Storage& aOtherStorage, const std::string& aCol
  * errors for columns occurring in both storage objects, and record the
  * values and column names in the comparisons and columnsUsed Arrays.
  */
-void Storage::compareWithStandard(Storage& standard, Array<string> &columnsUsed, Array<double> &comparisons)
+void Storage::compareWithStandard(Storage& standard, std::vector<string> &columnsUsed, std::vector<double> &comparisons)
 {
     int maxColumns = _columnLabels.getSize();
-    columnsUsed.ensureCapacity(maxColumns);
-    comparisons.ensureCapacity(maxColumns);
+    columnsUsed.reserve(maxColumns);
+    comparisons.reserve(maxColumns);
 
```

Comment:
No need to reserve space. This kind of thing is only helps with huge sized arrays of the order of 1000s or elements.